changelog.d/568.change.rst

```diff
@@ -0,0 +1 @@
+The value passed to ``@attr.ib(repr=…)`` can now be either a boolean (as before) or a custom callable, which makes it possible to influence how the attribute value is formatted by the generated ``__repr__()_`` method.
```

Comment:
You have an extra `_` after `__repr__()`

Reply:
fixed in b18eb9e6ef3cdc264b5461e168e3329a13dcce16